gettsim/benefits/elterngeld.py

```diff
@@ -0,0 +1,30 @@
+def elt_geld(df):
+    """This function calculates the monthly benefits for having
+    a child that is up to one year old (Elterngeld)"""
+
+    eltg_df = df
+
+    eltg_df["elt_geld"] = 0
+
+    receiving = eltg_df.loc[eltg_df["elt_zeit"], "pid"]
+
+    elt_geld = []
+
+    for pid in receiving:
+        referece_year = (
+            eltg_df.loc[(eltg_df["elt_zeit"] == True) & (eltg_df["pid"] == pid), "year"]
+            - 1
+        )
+        elt_geld += list(
+            eltg_df.loc[
+                (eltg_df["pid"] == pid)
+                & (eltg_df["year"] == referece_year.to_numpy()[0]),
+                "m_wage",
+            ]
+            * 2
+            / 3
+        )
```

Comment:
No hard coded numbers please. Parameters should be taken from param.xls (which is yet an excel file) and written into the `tb` dictionary. If I'm not mistaken, the parameters are already in the file.